webring.json

```diff
@@ -9,6 +9,10 @@
 		{
 			"name": "john",
 			"link": "johncarlomanuel.com"
+		},
+		{
+			"name":"matthew",
+			"link": "https://portfolio-rust-nine-27.vercel.app/"
```

Comment:
Remove `https://` and the trailing slash at the end of the link, then I can merge it!

Reply:
sounds good